lib/spack/spack/test/llnl/util/tty/log.py

```diff
@@ -19,7 +19,7 @@
 import pytest
 
 import llnl.util.tty.log
-from llnl.util.lang import uniq
+from llnl.util.lang import nullcontext, uniq
```

Comment:
Nitpick comment, but in general I would be for fully qualified namespaces instead of importing names from sub-sub-modules. The reason is that fully qualified namespaces are less subject to "circular-dependency" kind of issues + they are a good indicator of where some refactor is needed (if we constantly have to fully qualify a long name everywhere, it likely means the function we use belongs to a "higher" level in the chain of modules).

Reply:
Ok, this is done.